src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  --bluecolor: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: "Helvetica", sans-serif;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: auto;
+  box-sizing: border-box;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  padding-top: 23px;
+  padding-bottom: 22px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: normal;
+  text-align: center;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  &:hover {
+    color: var(--bluecolor);
+  }
+  &:active {
+    color: var(--bluecolor);
+    border-bottom: var(--bluecolor) solid 4px;
+    border-radius: 1px;
+  }
+}
+
+.is-active {
+  color: var(--bluecolor);
+  border-bottom: var(--bluecolor) solid 4px;
+}
```

Comment:
The line below active link should be styled with pseudo-elements, read about it in the readme and checklist.